### PR TITLE
Modify controlnet inferer to pass the same conditioning as the one th…

### PR DESCRIPTION
**Pull request by virginiafdez**
…e diffusion model is getting.

Modification of tests accordingly.

### CODE REVIEW

File: tests/test_controlnet_inferers.py [hunk 1]
```diff
@@ -15,8 +15,8 @@
 
 import torch
 from parameterized import parameterized
-
-from generative.inferers import ControlNetDiffusionInferer, ControlNetLatentDiffusionInferer
+from generative.networks.schedulers import DDIMScheduler
```
Comment (marksgraham) [inline tests/test_controlnet_inferers.py:18]: could you re-run autoformatting? these imports are unordered now

File: tests/test_controlnet_inferers.py [hunk 2]
```diff
@@ -537,8 +537,8 @@ def test_ddim_sampler(self, model_params, controlnet_params, input_shape):
 
     @parameterized.expand(CNDM_TEST_CASES)
     def test_sampler_conditioned(self, model_params, controlnet_params, input_shape):
-        model_params["with_conditioning"] = True
-        model_params["cross_attention_dim"] = 3
+        model_params["with_conditioning"] = controlnet_params["with_conditioning"] =  True
```
Comment (marksgraham) [inline tests/test_controlnet_inferers.py:540]: this looks like an error, two = operators?

File: tests/test_controlnet_inferers.py [hunk 3]
```diff
@@ -603,10 +603,12 @@ def test_normal_cdf(self):
     def test_sampler_conditioned_concat(self, model_params, controlnet_params, input_shape):
         # copy the model_params dict to prevent from modifying test cases
         model_params = model_params.copy()
+        controlnet_params = controlnet_params.copy()
         n_concat_channel = 2
         model_params["in_channels"] = model_params["in_channels"] + n_concat_channel
-        model_params["cross_attention_dim"] = None
-        model_params["with_conditioning"] = False
+        controlnet_params["in_channels"] = controlnet_params["in_channels"] + n_concat_channel
+        model_params["cross_attention_dim"] = controlnet_params["cross_attention_dim"] = None
```
Comment (marksgraham) [inline tests/test_controlnet_inferers.py:610]: here too

File: tests/test_controlnet_inferers.py [hunk 4]
```diff
@@ -975,24 +977,27 @@ def test_prediction_shape_conditioned_concat(
         autoencoder_params,
         dm_model_type,
         stage_2_params,
-        controlnet_params,
+        cn_params,
```
Comment (marksgraham) [inline tests/test_controlnet_inferers.py:980]: what was the motivation behind renaming controlnet_params to cn_params? if its to become consistent with usage elsewhere that seems fair, but if not, i think it just makes it a bit less readable

**Review by marksgraham — CHANGES_REQUESTED** [review]

File: tests/test_controlnet_inferers.py (hunk 2 above)
Comment (virginiafdez) [reply tests/test_controlnet_inferers.py:540]: What is the problem with this?

**Review by marksgraham — APPROVED** [review]

**Pull request merged**